sklearnex/preview/covariance/covariance.py

```diff
@@ -0,0 +1,96 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+from scipy import sparse as sp
+from sklearn.covariance import EmpiricalCovariance as sklearn_EmpiricalCovariance
+from sklearn.utils import check_array
+
+from daal4py.sklearn._utils import sklearn_check_version
+from onedal.common.hyperparameters import get_hyperparameters
+from onedal.covariance import EmpiricalCovariance as onedal_EmpiricalCovariance
+
+from ..._device_offload import dispatch
+from ..._utils import PatchingConditionsChain, register_hyperparameters
+
+
+@register_hyperparameters({"fit": get_hyperparameters("covariance", "compute")})
+class EmpiricalCovariance(sklearn_EmpiricalCovariance):
```

Comment:
Comment about `n_jobs` support is still valid: if one estimator is excluded, then extension will lose consistency.

Reply:
This request of changes is not critical.
We are agile. We are trying incrementally enable features in our library.
Support of n_jobs added after the estimators support.
Let's first have this pretty good PR merge, and then add n_jobs support in the next PR.